### ISSUE TITLE
Larger video size & bad quality compared to Odysee manual download

### ISSUE THREAD

**Issue by progonkpa** [yt-dlp/yt-dlp#4569]
### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.07.18** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

# Larger video size & bad quality compared to Odysee manual download

## General comparison

[Video of observation](https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3)

|Odyesee|yt-dlp|
|------------|--------|
|2.2GB|4.4GB|
|good quality|*bad quality|

* Still pictures have good quality but transitions that fade into each other contain too much block artifacts.

## Mediainfo comparison

After analyzing the mediainfo, I think this can be fixed by changing the X264/X265 profile from baseline to high. CABAC also seems important to be active. I think CABAC is part of high profile and enabled by default but not sure.

### Odysee

```
General
Complete name                            : /home/freetalk/Downloads/Europa - The Last Battle - Full Documentary.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/iso2/avc1/mp41)
File size                                : 2.09 GiB
Duration                                 : 11 h 59 min
Overall bit rate                         : 416 kb/s
Writing application                      : Lavf56.40.101

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L3.1
Format settings                          : CABAC / 4 Ref Frames
Format settings, CABAC                   : Yes
Format settings, Reference frames        : 4 frames
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 11 h 59 min
Bit rate                                 : 311 kb/s
Width                                    : 854 pixels
Height                                   : 480 pixels
Display aspect ratio                     : 16:9
Original display aspect ratio            : 16:9
Frame rate mode                          : Constant
Frame rate                               : 29.970 (30000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.025
Stream size                              : 1.56 GiB (75%)
Writing library                          : x264 core 148 r2643 5c65704
Encoding settings                        : cabac=1 / ref=2 / deblock=1:0:0 / analyse=0x3:0x113 / me=hex / subme=4 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=0 / threads=6 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=3 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=1 / keyint=120 / keyint_min=12 / scenecut=40 / intra_refresh=0 / rc_lookahead=20 / rc=crf / mbtree=1 / crf=30.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / ip_ratio=1.40 / aq=1:1.00
Language                                 : English
Codec configuration box                  : avcC

Audio
ID                                       : 2
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : mp4a-40-2
Duration                                 : 11 h 59 min
Duration_LastFrame                       : -9 ms
Bit rate mode                            : Constant
Bit rate                                 : 96.0 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 46.875 FPS (1024 SPF)
Compression mode                         : Lossy
Stream size                              : 494 MiB (23%)
Language                                 : English
Default                                  : Yes
Alternate group                          : 1

```

### yt-dlp

```
General
Complete name                            : /data/Users/krist/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/iso2/avc1/mp41)
File size                                : 4.06 GiB
Duration                                 : 11 h 59 min
Overall bit rate mode                    : Variable
Overall bit rate                         : 808 kb/s
Writing application                      : Lavf59.16.100

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : Baseline@L3.1
Format settings                          : 1 Ref Frames
Format settings, CABAC                   : No
Format settings, Reference frames        : 1 frame
Format settings, GOP                     : M=1, N=100
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 11 h 59 min
Source duration                          : 11 h 59 min
Bit rate                                 : 635 kb/s
Maximum bit rate                         : 667 kb/s
Width                                    : 854 pixels
Height                                   : 480 pixels
Display aspect ratio                     : 16:9
Original display aspect ratio            : 16:9
Frame rate mode                          : Constant
Frame rate                               : 29.970 (30000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.052
Stream size                              : 3.19 GiB (79%)
Source stream size                       : 3.35 GiB (83%)
Writing library                          : x264 core 164 r3079 d9a19f0
Encoding settings                        : cabac=0 / ref=1 / deblock=0:0:0 / analyse=0:0 / me=dia / subme=0 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=0 / 8x8dct=0 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=0 / threads=2 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=0 / weightp=0 / keyint=100 / keyint_min=10 / scenecut=0 / intra_refresh=0 / rc_lookahead=0 / rc=crf / mbtree=0 / crf=26.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=703 / vbv_bufsize=1406 / crf_max=0.0 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=0
Codec configuration box                  : avcC

Audio
ID                                       : 2
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : mp4a-40-2
Duration                                 : 11 h 59 min
Bit rate mode                            : Variable
Bit rate                                 : 132 kb/s
Maximum bit rate                         : 138 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 44.1 kHz
Frame rate                               : 43.066 FPS (1024 SPF)
Compression mode                         : Lossy
Stream size                              : 678 MiB (16%)
Language                                 : English
Default                                  : Yes
Alternate group                          : 1
```

### diff

The diff from the two mediainfo ouputs above, from odysee to yt-dlp. The same diff in color is to be found after the logs.

```
diff --git a/data/code/vidhop/docs/mediainfo downloaded file Odysee.md b/data/code/vidhop/docs/mediainfo downloaded file yt-dlp.md
index 6f386ad..d80fcbb 100644
--- a/data/code/vidhop/docs/mediainfo downloaded file Odysee.md	
+++ b/data/code/vidhop/docs/mediainfo downloaded file yt-dlp.md	
@@ -1 +1 @@
-## mediainfo downloaded file Odysee
+## mediainfo downloaded file yt-dlp
@@ -3,2 +3,2 @@
- General
-Complete name                            : /home/freetalk/Downloads/Europa - The Last Battle - Full Documentary.mp4
+General
+Complete name                            : /data/Users/krist/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4
@@ -8 +8 @@ Codec ID                                 : isom (isom/iso2/avc1/mp41)
-File size                                : 2.09 GiB
+File size                                : 4.06 GiB
@@ -10,2 +10,3 @@ Duration                                 : 11 h 59 min
-Overall bit rate                         : 416 kb/s
-Writing application                      : Lavf56.40.101
+Overall bit rate mode                    : Variable
+Overall bit rate                         : 808 kb/s
+Writing application                      : Lavf59.16.100
@@ -17,4 +18,5 @@ Format/Info                              : Advanced Video Codec
-Format profile                           : High@L3.1
-Format settings                          : CABAC / 4 Ref Frames
-Format settings, CABAC                   : Yes
-Format settings, Reference frames        : 4 frames
+Format profile                           : Baseline@L3.1
+Format settings                          : 1 Ref Frames
+Format settings, CABAC                   : No
+Format settings, Reference frames        : 1 frame
+Format settings, GOP                     : M=1, N=100
@@ -24 +26,3 @@ Duration                                 : 11 h 59 min
-Bit rate                                 : 311 kb/s
+Source duration                          : 11 h 59 min
+Bit rate                                 : 635 kb/s
+Maximum bit rate                         : 667 kb/s
@@ -35,5 +39,5 @@ Scan type                                : Progressive
-Bits/(Pixel*Frame)                       : 0.025
-Stream size                              : 1.56 GiB (75%)
-Writing library                          : x264 core 148 r2643 5c65704
-Encoding settings                        : cabac=1 / ref=2 / deblock=1:0:0 / analyse=0x3:0x113 / me=hex / subme=4 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=0 / threads=6 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=3 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=1 / keyint=120 / keyint_min=12 / scenecut=40 / intra_refresh=0 / rc_lookahead=20 / rc=crf / mbtree=1 / crf=30.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / ip_ratio=1.40 / aq=1:1.00
-Language                                 : English
+Bits/(Pixel*Frame)                       : 0.052
+Stream size                              : 3.19 GiB (79%)
+Source stream size                       : 3.35 GiB (83%)
+Writing library                          : x264 core 164 r3079 d9a19f0
+Encoding settings                        : cabac=0 / ref=1 / deblock=0:0:0 / analyse=0:0 / me=dia / subme=0 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=0 / 8x8dct=0 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=0 / threads=2 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=0 / weightp=0 / keyint=100 / keyint_min=10 / scenecut=0 / intra_refresh=0 / rc_lookahead=0 / rc=crf / mbtree=0 / crf=26.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=703 / vbv_bufsize=1406 / crf_max=0.0 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=0
@@ -48,3 +52,3 @@ Duration                                 : 11 h 59 min
-Duration_LastFrame                       : -9 ms
-Bit rate mode                            : Constant
-Bit rate                                 : 96.0 kb/s
+Bit rate mode                            : Variable
+Bit rate                                 : 132 kb/s
+Maximum bit rate                         : 138 kb/s
@@ -53,2 +57,2 @@ Channel layout                           : L R
-Sampling rate                            : 48.0 kHz
-Frame rate                               : 46.875 FPS (1024 SPF)
+Sampling rate                            : 44.1 kHz
+Frame rate                               : 43.066 FPS (1024 SPF)
@@ -56 +60 @@ Compression mode                         : Lossy
-Stream size                              : 494 MiB (23%)
+Stream size                              : 678 MiB (16%)
@@ -59 +63 @@ Default                                  : Yes
-Alternate group                          : 1
\ No newline at end of file
+Alternate group                          : 1
```

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['--dump-single-json', '--verbose', 'https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8 (No ANSI), error utf-8, screen utf-8
[debug] yt-dlp version 2022.07.18 [135f05e]
[debug] Python 3.10.5 (CPython 64bit) - Linux-5.16.20-2-MANJARO-x86_64-with-glibc2.35 (glibc 2.35)
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.12.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [lbry] Extracting URL: https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading stream JSON metadata
[lbry] 33246eb5291523d56894ccce142e405db960873d: Downloading streaming url JSON metadata
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading streaming redirect url info
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 33246eb5291523d56894ccce142e405db960873d: Downloading 1 format(s): hls-914
[debug] Command-line config: ['--no-download', '--print', '%(title)s', '--replace-in-metadata', 'title', '"', "'", '--replace-in-metadata', 'title', '\\*', '', '--replace-in-metadata', 'title', '/', '', '--replace-in-metadata', 'title', ':', ' -', '--replace-in-metadata', 'title', '<', '', '--replace-in-metadata', 'title', '>', '', '--replace-in-metadata', 'title', '\\?', '', '--replace-in-metadata', 'title', '\\\\', '', '--replace-in-metadata', 'title', '\\|', '-', '--replace-in-metadata', 'title', '\\`', "'", '--verbose', 'https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8 (No ANSI), error utf-8, screen utf-8
[debug] yt-dlp version 2022.07.18 [135f05e]
[debug] Python 3.10.5 (CPython 64bit) - Linux-5.16.20-2-MANJARO-x86_64-with-glibc2.35 (glibc 2.35)
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.12.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [lbry] Extracting URL: https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading stream JSON metadata
[lbry] 33246eb5291523d56894ccce142e405db960873d: Downloading streaming url JSON metadata
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading streaming redirect url info
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Replacing all '"' in title with "'"
[MetadataParser] Did not find '"' in title
[debug] Replacing all '\\*' in title with ''
[MetadataParser] Did not find '\\*' in title
[debug] Replacing all '/' in title with ''
[MetadataParser] Did not find '/' in title
[debug] Replacing all ':' in title with ' -'
[MetadataParser] Changed title to: Europa - The Last Battle - Full Documentary
[debug] Replacing all '<' in title with ''
[MetadataParser] Did not find '<' in title
[debug] Replacing all '>' in title with ''
[MetadataParser] Did not find '>' in title
[debug] Replacing all '\\?' in title with ''
[MetadataParser] Did not find '\\?' in title
[debug] Replacing all '\\\\' in title with ''
[MetadataParser] Did not find '\\\\' in title
[debug] Replacing all '\\|' in title with '-'
[MetadataParser] Did not find '\\|' in title
[debug] Replacing all '\\`' in title with "'"
[MetadataParser] Did not find '\\`' in title
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 33246eb5291523d56894ccce142e405db960873d: Downloading 1 format(s): hls-914
[debug] Command-line config: ['--write-thumbnail', '--convert-thumbnails', 'jpg', '--no-download', '-P', '/home/freetalk/Videos/DLV/metadata/videos/odysee', '-o', 'Europa - The Last Battle - Full Documentary.%(ext)s', '--verbose', 'https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.07.18 [135f05e]
[debug] Python 3.10.5 (CPython 64bit) - Linux-5.16.20-2-MANJARO-x86_64-with-glibc2.35 (glibc 2.35)
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.12.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [lbry] Extracting URL: https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading stream JSON metadata
[lbry] 33246eb5291523d56894ccce142e405db960873d: Downloading streaming url JSON metadata
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading streaming redirect url info
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 33246eb5291523d56894ccce142e405db960873d: Downloading 1 format(s): hls-914
Deleting existing file /home/freetalk/Videos/DLV/metadata/videos/odysee/Europa - The Last Battle - Full Documentary.jpg
[info] Downloading video thumbnail 0 ...
[info] Writing video thumbnail 0 to: /home/freetalk/Videos/DLV/metadata/videos/odysee/Europa - The Last Battle - Full Documentary.jpg
[ThumbnailsConvertor] Correcting thumbnail "/home/freetalk/Videos/DLV/metadata/videos/odysee/Europa - The Last Battle - Full Documentary.jpg" extension to webp
[ThumbnailsConvertor] Not converting thumbnail "/home/freetalk/Videos/DLV/metadata/videos/odysee/Europa - The Last Battle - Full Documentary.jpg"; already is in target format jpg
[debug] Command-line config: ['-S', 'ext:mp4:m4a', '--recode', 'mp4', '--no-mtime', '-P', '/home/freetalk/Videos/DLV/videos/odysee', '-o', 'Europa - The Last Battle - Full Documentary.%(ext)s', '--verbose', 'https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.07.18 [135f05e]
[debug] Python 3.10.5 (CPython 64bit) - Linux-5.16.20-2-MANJARO-x86_64-with-glibc2.35 (glibc 2.35)
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 5.0.1 (setts), ffprobe 5.0.1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.12.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] [lbry] Extracting URL: https://odysee.com/@truthte11er2.0:d/Europa-The-Last-Battle-Full-Documentary:3
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading stream JSON metadata
[lbry] 33246eb5291523d56894ccce142e405db960873d: Downloading streaming url JSON metadata
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading streaming redirect url info
[lbry] @truthte11er2.0#d/Europa-The-Last-Battle-Full-Documentary#3: Downloading m3u8 information
[debug] Sort order given by user: ext:mp4:m4a
[debug] Formats sorted by: hasvid, ie_pref, vext:mp4(5), aext:m4a(8), lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 33246eb5291523d56894ccce142e405db960873d: Downloading 1 format(s): hls-914
[debug] Invoking hlsnative downloader on "https://player.odycdn.com/api/v4/streams/tc/Europa-The-Last-Battle-Full-Documentary/33246eb5291523d56894ccce142e405db960873d/e20f9ece178de2e4a0f2a9bea29c9da5f872536e7c32f32d63b103959d37c97b16892e4cb18839f6458343574dbdc6ad/stream_1.m3u8"
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 4318
[download] Destination: /home/freetalk/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4
[download]  41.4% of ~4.33GiB at    2.31MiB/s ETA 24:26 (frag 1789/4318)[download] Got server HTTP error: The read operation timed out. Retrying (attempt 1 of 10) ...
[download] 100% of 4.29GiB in 01:45:31
[debug] ffprobe command line: ffprobe -hide_banner -show_format -show_streams -print_format json 'file:/home/freetalk/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4'
[FixupM3u8] Fixing MPEG-TS in MP4 container of "/home/freetalk/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:/home/freetalk/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4' -map 0 -dn -ignore_unknown -c copy -f mp4 -bsf:a aac_adtstoasc -movflags +faststart 'file:/home/freetalk/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.temp.mp4'
[VideoConvertor] Not converting media file "/home/freetalk/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4"; already is in target format mp4
```

### Mediainfo diff --color (use cat)

```
[1mdiff --git a/data/code/vidhop/docs/mediainfo downloaded file Odysee.md b/data/code/vidhop/docs/mediainfo downloaded file yt-dlp.md[m
[1mindex 6f386ad..d80fcbb 100644[m
[1m--- a/data/code/vidhop/docs/mediainfo downloaded file Odysee.md[m	
[1m+++ b/data/code/vidhop/docs/mediainfo downloaded file yt-dlp.md[m	
[36m@@ -1 +1 @@[m
[31m-## mediainfo downloaded file Odysee[m
[32m+[m[32m## mediainfo downloaded file yt-dlp[m
[36m@@ -3,2 +3,2 @@[m
[31m- General[m
[31m-Complete name                            : /home/freetalk/Downloads/Europa - The Last Battle - Full Documentary.mp4[m
[32m+[m[32mGeneral[m
[32m+[m[32mComplete name                            : /data/Users/krist/Videos/DLV/videos/odysee/Europa - The Last Battle - Full Documentary.mp4[m
[36m@@ -8 +8 @@[m [mCodec ID                                 : isom (isom/iso2/avc1/mp41)[m
[31m-File size                                : 2.09 GiB[m
[32m+[m[32mFile size                                : 4.06 GiB[m
[36m@@ -10,2 +10,3 @@[m [mDuration                                 : 11 h 59 min[m
[31m-Overall bit rate                         : 416 kb/s[m
[31m-Writing application                      : Lavf56.40.101[m
[32m+[m[32mOverall bit rate mode                    : Variable[m
[32m+[m[32mOverall bit rate                         : 808 kb/s[m
[32m+[m[32mWriting application                      : Lavf59.16.100[m
[36m@@ -17,4 +18,5 @@[m [mFormat/Info                              : Advanced Video Codec[m
[31m-Format profile                           : High@L3.1[m
[31m-Format settings                          : CABAC / 4 Ref Frames[m
[31m-Format settings, CABAC                   : Yes[m
[31m-Format settings, Reference frames        : 4 frames[m
[32m+[m[32mFormat profile                           : Baseline@L3.1[m
[32m+[m[32mFormat settings                          : 1 Ref Frames[m
[32m+[m[32mFormat settings, CABAC                   : No[m
[32m+[m[32mFormat settings, Reference frames        : 1 frame[m
[32m+[m[32mFormat settings, GOP                     : M=1, N=100[m
[36m@@ -24 +26,3 @@[m [mDuration                                 : 11 h 59 min[m
[31m-Bit rate                                 : 311 kb/s[m
[32m+[m[32mSource duration                          : 11 h 59 min[m
[32m+[m[32mBit rate                                 : 635 kb/s[m
[32m+[m[32mMaximum bit rate                         : 667 kb/s[m
[36m@@ -35,5 +39,5 @@[m [mScan type                                : Progressive[m
[31m-Bits/(Pixel*Frame)                       : 0.025[m
[31m-Stream size                              : 1.56 GiB (75%)[m
[31m-Writing library                          : x264 core 148 r2643 5c65704[m
[31m-Encoding settings                        : cabac=1 / ref=2 / deblock=1:0:0 / analyse=0x3:0x113 / me=hex / subme=4 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=0 / threads=6 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=3 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=1 / keyint=120 / keyint_min=12 / scenecut=40 / intra_refresh=0 / rc_lookahead=20 / rc=crf / mbtree=1 / crf=30.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / ip_ratio=1.40 / aq=1:1.00[m
[31m-Language                                 : English[m
[32m+[m[32mBits/(Pixel*Frame)                       : 0.052[m
[32m+[m[32mStream size                              : 3.19 GiB (79%)[m
[32m+[m[32mSource stream size                       : 3.35 GiB (83%)[m
[32m+[m[32mWriting library                          : x264 core 164 r3079 d9a19f0[m
[32m+[m[32mEncoding settings                        : cabac=0 / ref=1 / deblock=0:0:0 / analyse=0:0 / me=dia / subme=0 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=0 / me_range=16 / chroma_me=1 / trellis=0 / 8x8dct=0 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=0 / threads=2 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=0 / weightp=0 / keyint=100 / keyint_min=10 / scenecut=0 / intra_refresh=0 / rc_lookahead=0 / rc=crf / mbtree=0 / crf=26.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=703 / vbv_bufsize=1406 / crf_max=0.0 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=0[m
[36m@@ -48,3 +52,3 @@[m [mDuration                                 : 11 h 59 min[m
[31m-Duration_LastFrame                       : -9 ms[m
[31m-Bit rate mode                            : Constant[m
[31m-Bit rate                                 : 96.0 kb/s[m
[32m+[m[32mBit rate mode                            : Variable[m
[32m+[m[32mBit rate                                 : 132 kb/s[m
[32m+[m[32mMaximum bit rate                         : 138 kb/s[m
[36m@@ -53,2 +57,2 @@[m [mChannel layout                           : L R[m
[31m-Sampling rate                            : 48.0 kHz[m
[31m-Frame rate                               : 46.875 FPS (1024 SPF)[m
[32m+[m[32mSampling rate                            : 44.1 kHz[m
[32m+[m[32mFrame rate                               : 43.066 FPS (1024 SPF)[m
[36m@@ -56 +60 @@[m [mCompression mode                         : Lossy[m
[31m-Stream size                              : 494 MiB (23%)[m
[32m+[m[32mStream size                              : 678 MiB (16%)[m
[36m@@ -59 +63 @@[m [mDefault                                  : Yes[m
[31m-Alternate group                          : 1[m
\ No newline at end of file[m
[32m+[m[32mAlternate group                          : 1[m

```

**Comment by progonkpa** [comment]
@pukkandan  After analyzing the mediainfo, I think this can be fixed by changing the X264 profile from baseline to high. CABAC also seems important to be active. I think CABAC is part of, and enabled by default, by high profile but not sure.

**Comment by bashonly** [comment]
Fixed by 127a22460658ac39cbe5c4b3fb88d578363e0dfa
